### DISCUSSION TITLE
Did UI change?

### DISCUSSION THREAD

**Discussion by madschristensen99** [oobabooga/text-generation-webui#3068] [General]
I noticed that there is no longer a "start response" with field which was vital for me to get sensible responses out of 7B models. What gives?

**Comment by viperwasp** [comment]
It's still there for me but I have not updated in 24 hours. I still have a Start reply with section down below clear history. 
When did you last update? Now I'm scared to update this is a vital feature. If it's removed Ooba is kind of not usable.

**Comment by reillyardstard** [comment]
the "start reply with" option appears if I open the webui in chat mode i.e. cmd_flags includes --chat
